scripts/qa_crowbarsetup.sh

```diff
@@ -2450,6 +2450,12 @@ function enable_ssl_generic()
             fi
             $p "$a['api']['protocol']" "'https'"
         ;;
+        manila)
+            if [[ ! $(iscloudver 7plus) ]] ; then
```

Comment:
@dirkmueller I basically did it the same as its done in the lines above for consistence, but indeed it makes no sense when it could be much cleaner

    if iscloudver 7plus ; then
      $p "$a['api']['protocol']" "'https'"
    fi

thoughts on consistence vs clarity?